src/tests/tablet/schedule/CreateMeetingIfNotHasBody.java

```diff
@@ -0,0 +1,69 @@
+package tests.tablet.schedule;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+import framework.pages.tablet.HomeTabletPage;
+import framework.pages.tablet.SchedulePage;
+import framework.rest.RootRestMethods;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC18: Verify a meeting can be created if the body is empty
+ * TC28: Verify that pressing enter confirms the name of an attendee
+ * TC29: Verify that Pressing semicolon confirms the name of an attendee
+ * @author Asael Calizaya
+ *
+ */
+public class CreateMeetingIfNotHasBody {
+	ExcelReader excelReader = new ExcelReader(EXCEL_INPUT_DATA);
+	List<Map<String, String>> meetingData = excelReader.getMapValues("MeetingData");
+	String organizer = meetingData.get(0).get("Organizer");
```

Comment:
Maybe we can coordinate about this to add in meeting methods the method that a I have about create meeting with Excel values...